### DISCUSSION TITLE
Economy: Server Account

### DISCUSSION THREAD

**Discussion by austin4195** [Zrips/CMI#3411] [Feature Requests]
**Description of feature/improvement request:**

The addition of having a server account for usage with Jobs reborn to handle taxes and payouts.

---
### Detailed explanation of what it should do, when and in what situation: ###

This feature would essentially create a dummy player with the name Server or Server Account that would allow for a "server bank" to control flow of in game currency.

### Basic explanation as to why this feature is needed and/or useful to have: ###

This can be useful for economy regulation by limiting how much money is permitted into the economy. It would also allow a staff member who's balance is currently serving the purpose of taxes to be able to enjoy the game. It would also allow functionality between Jobs Reborn and CMI Economy as it seems that the server account function was added for every other economy plugin except this one.

**Comment by KoertLichtendonk** [comment]
Supporting, I just realized that this is missing so I can't use the starting balance as I want a circular/closed economy.
